src/btree/row_modify.c

```diff
@@ -347,6 +349,7 @@ __wt_update_obsolete_check(
                 first = upd;
         } else {
             first = NULL;
+            min_ts = upd->durable_ts;
```

Comment:
Although we expect the update to be monotonically increasing in terms of commit timestamp, we still support committing out-of-order update. Can we make this check something like this:
`min_ts = WT_MIN(upd->durable_ts, min_ts)`

Reply:
I am assuming the intent here is to taking the minimum of all the durable timestamps on the update chain. I would like to see more comments here.

Reply:
Done